src/Hedgehog/Gen.fs

```diff
@@ -354,7 +368,7 @@ module Gen =
 
     /// Generates a random string.
     let string : Gen<string> =
-        choice [alphaNum; charBounded] |> string'
+        choice [alphaNum; unicodeAll] |> string'
```

Comment:
I don't think there is a haskell version, the haskell `Gen.string` takes a `Gen<char>` as an argument